website/organizations/index.html

```diff
@@ -22,14 +22,79 @@ <h3 class="header-section">
+
+        <h3 class="subheader-section">
+          {{ _('Stability, efficiency, and security for your emails') }}
+        </h3>
+        <p class="font-lg tracking-wider leading-loose mb-10 mt-0 p-links-blue">
+          {{ _('Provide your team with a stable communications solution which allows them to focus on getting their work done. The <a href="/">regular Thunderbird release</a> follows the so-called <a href="{firefox_release_calendar}">Extended Support Release (ESR) schedule</a> of the Mozilla platform. So to your advantage it will remain stable and feature-stable on that major version for approximately one year. Of course, we&#39;ll always keep you on the safe side with monthly minor upgrades which may contain stability, correctness and security upgrades.')|f(firefox_release_calendar=url('firefox.release-calendar')) }}
         </p>
+
+        <h3 class="subheader-section">
+          {{ _('Get Thunderbird for your organization') }}
+        </h3>
         <p class="font-lg tracking-wider leading-loose mb-10 mt-0 p-links-blue">
-          {{ _('We do have other resources for organizations: If you need it, you can find the Thunderbird MSI Installer on the <a href="{syslang}">Systems & Languages page</a>. For more information about deploying and configuring Thunderbird in business and enterprise environments you can also join the <a href="{maillist}">Enterprise mailing list</a>.')|f(maillist=url('thunderbird.enterprise'), syslang=url('thunderbird.latest.all')) }}
+          {{ _('Thunderbird is available in a <a href="{syslang}">variety of languages</a> for all major operating systems (Windows / MacOS / Linux). For multi-lingual environments, you can download language pack add-ons and switch Thunderbird's user interface language on the fly. We offer a number of resources specifically for organizations:')|f(syslang=url('thunderbird.latest.all')) }}
         </p>
+
+        <ul class="font-md tracking-wider leading-loose mb-6 p-links-blue">
+          <li>
+            {{ _('For Windows deployment, you can find the Thunderbird MSI Installer on the <a href="{syslang}">Systems &amp; Languages page</a>.')|f(syslang=url('thunderbird.latest.all')) }}
+          </li>
+          <li>
+            {{ _('For more information about deploying and configuring Thunderbird in business and enterprise environments, please visit our enterprise deployment portal: <a href="{thunderbird_etn}">enterprise.thunderbird.net</a>.')|f(thunderbird_etn=url('thunderbird.enterprise.documentation')) }}
+          </li>
+          <li>
+            {{ _('You can also join the <a href="{tb_list_enterprise}">Thunderbird Enterprise mailing list on Topicbox</a> to connect with other administrators and share advice on Thunderbird deployment (Please note: The <a href="tb_oldlist_enterprise">old tb-enterprise mailing list</a> has been retired).')|f(tb_list_enterprise=url('thunderbird.lists.enterprise'), tb_oldlist_enterprise=url('thunderbird.oldlists.enterprise')) }}
```

Comment:
Do we really want to be linking to the old mailing lists? Is there important stuff people might need to search for in there? I feel like we shouldn't be mentioning outdated resources.

Reply:
Afasik there IS important stuff in there, but maybe you're right, this isn't the right place to refer to collections of  wisdom from the past.